clients/src/test/java/org/apache/kafka/common/requests/LeaveGroupRequestTest.java

```diff
@@ -57,4 +120,9 @@ public void testLeaveConstructor() {
                                                                     Errors.NOT_CONTROLLER.exception()));
         }
     }
+
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
One more test case is using multiple members on an old version of the api.

Reply:
I guess it's covered in `testMultiLeaveConstructor`?